Feat/264 - Landing page "About" section

vogelino opened the pull request:
This PR adds the "About" section of the landing page, like in the designs.

dnsos commented on src/components/LandingLabAbout/index.tsx:
```diff
@@ -0,0 +1,113 @@
+          {furtherProjects.map(({ name, repo, logo }) => (
+            <a
+              key={name}
+              href={`https://github.com/technologiestiftung/`}
```
Here we wanna link to the actual `repo`, right?

vogelino replied:
Right! There is a bug in VS Code to be aware of: When you use the command "Wrap with an element" it removes the variables used in template strings. This is what happened here. Good catch!

dnsos commented on src/components/LandingLabAbout/index.tsx:
```diff
@@ -0,0 +1,113 @@
+            <a
+              key={name}
+              href={`https://github.com/technologiestiftung/`}
+              title={`Das Code-Repository von `}
```
Add the `name` here

vogelino replied:
Same thing here. I will insert it back

dnsos commented on src/components/LandingProjectAbout/index.tsx:
```diff
@@ -0,0 +1,121 @@
+                    <a
+                      href={link}
+                      title={description}
+                      className='text-gray-500 underline hover:bg-blue-500 transition'
```
The hover style looks weird with the gray text on blue background. Maybe that was actually supposed to mean `hover:text-blue-500`?

vogelino replied:
I will try it out! Good idea!

dnsos commented on src/components/LandingLabAbout/index.tsx:
```diff
@@ -0,0 +1,113 @@
+            "sm:border-t-0 border-gray-200",
+          ].join(" ")}
+        >
+          <strong className='font-bold'>226</strong> Open Source Projekte
```
At some point we should fetch this number from the GitHub API, but totally fine for now as some texts are actually still placeholders anyway

vogelino replied:
I agree! I will look into how to do that!

dnsos reviewed (changes requested):
Amazing work, well done @vogelino!

There are just a few small things that may need to be addressed, see my comments for details.

vogelino commented:
> There are just a few small things that may need to be addressed, see my comments for details.

Thank you @dnsos for your good review! I fixed the template literals and added a hook to fetch the number of repos dynamically of the homepage about section. Check it out! :)

dnsos reviewed (approved):
Amazing, now let's merge this!

Great that you already implemented the dynamic number of repos btw 🚀

tsboter commented:
:tada: This PR is included in version 1.0.0 :tada:

The release is available on [GitHub release](https://github.com/technologiestiftung/stadtpuls-frontend/releases/tag/v1.0.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
